client/pages/quiz/[slug].js

```diff
@@ -0,0 +1,166 @@
+/*
+  This page will load at the url "/quiz/:slug"
+*/
+
+import React, { useState, useEffect } from "react";
+// import useSWR from 'swr';// <-- uncomment when DB is ready
+import { useRouter } from "next/router";
+// import apiRoutes from '../../api-routes';// <-- uncomment when DB is ready
+import { dummyData } from "../dummy-quiz-data";
+// import DisplayMessage from '../../components/shared/DisplayMessage';
```

Comment:
Needs same comment as above, if it's commented out due to missing DB

Reply:
i added missing comments